src/clj/rems/db/blacklist.clj

```diff
@@ -53,6 +53,7 @@
 
 (defn get-events [& [params]]
   (->> (vals (cache/entries! blacklist-event-cache))
+       (sort-by :event/id)
```

Comment:
`rems.db.events` uses `sorted-map` to maintain the ordering

Reply:
Good point, let's use the same here as well